APMrover2/Parameters.h

```diff
@@ -43,8 +43,8 @@ class Parameters {
         k_param_battery_curr_pin,
 
         // braking
-        k_param_braking_percent = 30,
-        k_param_braking_speederr,
+        k_param_braking_percent = 30,	// unused
```

Comment:
Often we append `_unused` to the variable name to ensure we've killed all of the uses of it.